React/todoApp/src/components/Item.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import styles from "./Item.module.css";
+
+function Item({ item, setTodos }) {
+  const [editing, setEditing] = React.useState(false);
+  const inputRef = React.useRef(null);
+
+  const toggleComplete = () => {
+    setTodos(prev =>
+      prev.map(t => (t.id === item.id ? { ...t, is_completed: !t.is_completed } : t))
+    );
+  };
+
+  const deleteTodo = () => {
+    setTodos(prev => prev.filter(t => t.id !== item.id));
```

Comment:
Just a simple change inside the lambda but makes world of changes when it comes to maintaining this software 2 or 20 years later

```suggestion
    setTodos(prev => prev.filter(todo => todo.id !== item.id));
```